sim/battle.ts

```diff
@@ -1884,6 +1884,7 @@ export class Battle extends Dex.ModdedDex {
 			} else {
 				this.add('-activate', target, 'Substitute', '[damage]');
 			}
+			this.hint('In Gen 1, Substitute also takes confusion and Jump Kick/Hi Jump Kick recoil damage.');
```

Comment:
```suggestion
			this.hint("In Gen 1, Substitute also takes confusion and Jump Kick/Hi Jump Kick recoil damage.");
```

Reply:
Removed this hint. Will add it back when the behavior is fixed (which I'll probably look to do today, but I dont like mixing functionality changes with hints)